src/engine/strat_engine/engine.rs

```diff
@@ -36,6 +37,8 @@ pub struct StratEngine {
 
 impl StratEngine {
     pub fn new() -> StratEngine {
+        let _ = create_dir("/dev/stratis");
```

Comment:
Probably better to match against the result. If it fails for any other reason than that it already exists, you probably have a problem. Also, probably better to change the name to ```initialize```, since it has, potentially, this big side-effect, and change the result type to EngineResult.That's been the general trend, recently.